src/commands/appdistribution-distribute.ts

```diff
@@ -72,11 +72,24 @@ module.exports = new Command("appdistribution:distribute <distribution-file>")
     const testers = getTestersOrGroups(options.testers, options.testersFile);
     const groups = getTestersOrGroups(options.groups, options.groupsFile);
     const requests = new AppDistributionClient(appId);
+    let app: AppDistributionApp;
 
     try {
-      await requests.provisionApp();
+      app = await requests.getApp();
     } catch (err) {
-      throw new FirebaseError(`failed to provision app with ${err.message}`, { exit: 1 });
+      throw new FirebaseError(
+        "your app could not be found, make sure to onboard your app with App " +
```

Comment:
If the linter isn't making you, don't split up the string into multiple lines with ` + `.

Can we make this error message something like:

    `App Distribution is not enabled for app ${appId}. Please visit the Firebase Console to get started: https://actual.console.link.they.would.need`

Reply:
The linter caused this. Do we have enough info in this context to build a console url?

Reply:
I think your reply above belonged in this thread :) Resolving.

Reply:
Please still change the wording as per my comment, just omit the URL at the end.

Reply:
Done.